Scheduler/observing_blocks_auxtel/BLOCK-296/BLOCK-296_g.json

```diff
@@ -0,0 +1,293 @@
+      "standard": true,
+      "parameters": {
+        "find_target": {
+          "el": 80,
+          "az": 0,
+          "mag_limit": 8.0,
+          "radius": 2
+        }
+      }
+    },
+    {
+      "name": "auxtel/latiss_take_sequence.py",
+      "standard": true,
+      "parameters": {
+        "program": "BLOCK-296",
+        "reason": "SITCOM-1375-g",
+        "exposure_time_sequence": [
+          30,
+          30,
+          30
+        ],
+        "filter_sequence": [
+          "SDSSg_65mm",
+          "SDSSg_65mm",
+          "SDSSg_65mm"
+        ],
+        "grating_sequence": [
+          "empty_1",
+          "empty_1",
+          "empty_1"
+        ]
+      }
+    },
+    {
+      "name": "auxtel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "find_target": {
+          "el": 50,
+          "az": 0,
+          "mag_limit": 8.0,
+          "radius": 2
+        }
+      }
+    },
+    {
+      "name": "auxtel/latiss_take_sequence.py",
+      "standard": true,
+      "parameters": {
+        "program": "BLOCK-296",
+        "reason": "SITCOM-1375-g",
+        "exposure_time_sequence": [
+          30,
+          30,
+          30
+        ],
+        "filter_sequence": [
+          "SDSSg_65mm",
+          "SDSSg_65mm",
+          "SDSSg_65mm"
+        ],
+        "grating_sequence": [
+          "empty_1",
+          "empty_1",
+          "empty_1"
+        ]
+      }
+    },
+    {
+      "name": "auxtel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "find_target": {
+          "el": 40,
+          "az": 0,
+          "mag_limit": 8.0,
+          "radius": 2
+        }
+      }
+    },
+    {
+      "name": "auxtel/latiss_take_sequence.py",
+      "standard": true,
+      "parameters": {
+        "program": "BLOCK-296",
+        "reason": "SITCOM-1375-g",
+        "exposure_time_sequence": [
+          30,
+          30,
+          30
+        ],
+        "filter_sequence": [
+          "SDSSg_65mm",
+          "SDSSg_65mm",
+          "SDSSg_65mm"
+        ],
+        "grating_sequence": [
+          "empty_1",
+          "empty_1",
+          "empty_1"
+        ]
+      }
+    },
+    {
+      "name": "auxtel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "find_target": {
+          "el": 30,
+          "az": 0,
+          "mag_limit": 8.0,
+          "radius": 2
+        }
+      }
+    },
+    {
+      "name": "auxtel/latiss_take_sequence.py",
+      "standard": true,
+      "parameters": {
+        "program": "BLOCK-296",
+        "reason": "SITCOM-1375-g",
+        "exposure_time_sequence": [
+          30,
+          30,
+          30
+        ],
+        "filter_sequence": [
+          "SDSSg_65mm",
+          "SDSSg_65mm",
+          "SDSSg_65mm"
+        ],
+        "grating_sequence": [
+          "empty_1",
+          "empty_1",
+          "empty_1"
+        ]
+      }
+    },
+    {
+      "name": "auxtel/track_target.py",
```

Comment:
Did you mean to repeat el=30 here? If so, you can skip the track_target part and just take 6 exposures.